File: uri 1020.java
import java.util.Scanner;
 

public class Main {
 
    public static void main(String[] args)  {
 
        Scanner input = new Scanner(System.in);
        int n,y,m,d;
        n = input.nextInt();
        y = n/365;
        n = n%365;
        m = n/30;
        d = n%30;
        System.out.printf("%d ano(s)\n%d mes(es)\n%d dia(s)\n",y,m,d);
        
        
    }
 
}

File: uri 1020.c
#include<stdio.h>
int main ()

{
    int N,y,m,d;
    scanf ("%d",&N);
    y = N / 365;
    N = N % 365;
    m = N / 30;
    d = N % 30;
    printf("%d ano(s)\n",y);
    printf("%d mes(es)\n",m);
    printf("%d dia(s)\n",d);
    return 0;




}
